### PR TITLE
feat: add vmware log insight fluentd output

### PR DESCRIPTION
**Pull request by logikone**
* add fluentd output plugin for VMware LogInsight

### CODE REVIEW

File: pkg/sdk/logging/model/output/vmware_loginsight.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+// +name:"VMware LogInsight"
+// +url:"https://github.com/vmware/fluent-plugin-vmware-loginsight/releases/tag/v1.4.2"
+// +version:"1.4.2"
+// +description:"Store logs in VMware LogInsight"
+// +status:"GA"
+type _metaVMwareLogInsight interface{} //nolint:deadcode,unused
+
+// +kubebuilder:object:generate=true
+// +docName:"VMwareLogInsight"
+// LogZ Send your logs to VNMware LogInsight
```
Comment (pepov) [inline pkg/sdk/logging/model/output/vmware_loginsight.go:59]: Is this a copy-paste issue perhaps?
```suggestion
// Send your logs to VNMware LogInsight
```

**Review by pepov — APPROVED** [review]
Nice job, thanks!

File: pkg/sdk/logging/model/output/vmware_loginsight.go (hunk 1 above)
Comment (logikone) [reply pkg/sdk/logging/model/output/vmware_loginsight.go:59]: ah, yes. i will get that fixed 👍

Comment (logikone) [reply pkg/sdk/logging/model/output/vmware_loginsight.go]: ok, this is fixed now. thank you!

**Review by OverOrion — APPROVED** [review]

**Pull request merged**